omod/src/main/resources/apps/overallActions_extension.json

```diff
@@ -29,5 +29,15 @@
         "icon": "icon-link",
         "order": 13,
         "requiredPrivilege": "Task: coreapps.mergeVisits"
+    },
+    {
+        "id": "${project.parent.groupId}.${project.parent.artifactId}.deletePatient",
+        "extensionPointId": "patientDashboard.overallActions",
+        "type": "script",
+        "label": "coreapps.task.deletePatient.label",
+        "script": "delPatient.showDeletePatientCreationDialog({{patient.uuid}})",
```

Comment:
@djazayeri @wluyima since the REST call requires the UUID, I am passing patient.uuid as parameter. However, when I click on the DeletePatient button, I get the error `Uncaught SyntaxError: missing ) after argument list`. Just to see if there was any other syntactical problem, I changed `patient.uuid` to `patient.patientId` and the button works fine. Obviously, the REST call won't since it needs the uuid. SO I am thinking there is a problem here

Another thing: Doesn't the REST call need to pass in 'Reason for deleting patient'? Can we get rid of that then? If we do it from the API then I think it is needed.